shell/platform/embedder/embedder.h

```diff
@@ -2263,6 +2263,8 @@ typedef struct {
 
 #ifndef FLUTTER_ENGINE_NO_PROTOTYPES
 
+// NOLINTBEGIN(google-objc-function-naming)
```

Comment:
Is there a way for us to not apply google-objc-function-naming to non-Objective-C headers?

We should apply it to headers under //flutter/shell/platform/darwin, but most of the repo should be treated as C/C++.

Reply:
My guess is because this is imported by Obj-C code, it assumes it's Obj-C.